packages/contracts/deploy/40_finalize-managing-dao/20_revoke-deployer-permissions-of-DAO.ts

```diff
@@ -0,0 +1,38 @@
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
```

Comment:
Same as above, I would do the whole process in 2 steps:  (1) managing DAO in isolation and (2) Framework deployment